apps/wakunode2/external_config.nim

```diff
@@ -576,6 +583,15 @@ proc parseCmdArg*(T: type Option[int], p: string): T =
   except CatchableError:
     raise newException(ValueError, "Invalid number")
 
+proc completeCmdArg*(T: type ShardIdx, val: string): seq[ShardIdx] =
+  return @[]
+
+proc parseCmdArg*(T: type ShardIdx, p: string): T =
+  try:
+    ShardIdx(parseInt(p))
```

Comment:
Would it be possible to ensure that the given index is within the valid boundaries?

Reply:
will address this in followup prs. don't want to just add a `MaxAmountShards` here but enforce it all over the place when a given cluster-id is chosen. and that may require some changes out of scope of this pr :)